TreeAPI.js

```diff
@@ -1,47 +1,87 @@
 /*
  * TreeAPI.js
- * 
+ *
  * Provide a "Person" object where data is gathered from the WikiTree API.
  * We use the WikiTree API action "getPerson" to retrieve the profile data and then store it in object fields.
- * 
+ *
  */
 
 
 // Put our functions into a "WikiTreeAPI" namespace.
 window.WikiTreeAPI = window.WikiTreeAPI || {};
 
 // Our basic constructor for a Person. We expect the "person" data from the API returned result
-// (see getPerson below). The basic fields are just stored in the internal _data array.
+// (see getPersonViaAPI below). The basic fields are just stored in the internal _data array.
 // We pull out the Parent and Child elements as their own Person objects.
 WikiTreeAPI.Person = function(data){
+	let self = this;
```

Comment:
Is there a reason to introduce this new `self` variable? I'm not really sure how it works with prototypes, but in class based objects, if you want to refer tl itself the `this` keyword is used. And it's already used in code this way. Btw `self` in javascript refers to `window` object.

https://stackoverflow.com/a/3217907/2512455

Reply:
No, it was just my paranoia/misunderstanding of something I read (that I can't find now - to do with the fact that in JavaScript, `this`  is not always referring to what one might think it does.).